### DISCUSSION TITLE
[Q&A] Represent Cis-Trans in SMILES Notation of Functional Groups in the Ring (No identifying bond type of ring)

### DISCUSSION THREAD

**Discussion by IchiruTake** [rdkit/rdkit#4593] [General] [answered]
I want to ask that is there any representation that can push SMILES support initial thought about functional group direction in ring. In particular, let's say that, Cyclo-hexane or benzene for example, and inserting functional groups (example: Methyl to be easy). In the SMILES Notation, I should expect it can somehow draw or represent partially the direction to be drawn even though the molecular fingerprint would be identical.

Cyclo-hexane (`CC1CCCCC1C`): 

(Z)-1,2-Me2 == `C/C1CCCCC1\C`

(E)-1,2-Me2 == `C/C1CCCCC1/C`

...

**Reference**: Compresive Handbook of Bond Dissociation Energies of Yu Ran Luo (2007) --> Page 35, 36

**Comment by greglandrum** [comment, marked as answer]
Hi, I hope I've understood the question correctly.
In this situation cis/trans isn't the correct terminology as far as the RDKit is concerned. Within the RDKit cis/trans applies to bonds, but here we're dealing with atomic stereochemistry: chirality
Are you asking how to distinguish between these two structures?
![image](https://user-images.githubusercontent.com/540511/137427283-bb832408-c014-45a9-b1b9-28d534d482c7.png)
